contents/handbook/content-and-docs/docs.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Docs ownerhship
+sidebar: Handbook
+showTitle: true
+---
+
+- **Product teams** are responsible for ensuring their docs are up-to-date. This means:
+  - Documenting new features when they're launched.
+  - Correcting mistakes reported by users.
+  - Clarifying documentation where needed based on support tickets.
+  - Ensuring public betas have documentation which is linked to from feature preview menu
+
+- **Content marketing** is responsible for improving the docs. This means:
+  - Reviewing and improving draft documentation created by product teams.
+  - Identifying and improving low quality documentation.
+  - Ensuring screenshots and other visual elements are up-to-date.
+  - Shipping supplementary docs and tutorials based on feedback and emerging use cases.
+
+- **Website & Docs** is responsible for design, organization, and discovery. This means:
+  - The design and content of index pages.
+  - The overall layout of docs and how they're organized.
+  - In-page elements and components – e.g. light and dark mode screenshots.
+  - Website search and other elements that help users find the answers they need.
+
+## Ownership in Content & Docs team
```

Comment:
@bijanbwb @ivanagas @daniloc @Lior539 I've "guestimated" ownership based on who who has worked most on a specific area of the docs in the past / knowledge, etc, but this is very rough. My main goal is just to add a little clarity internally, both for us and others.

Feel free to suggest changes, add / remove and so on. I imagine this will evolve a lot over time, this is just a first attempt to get something written down.

Reply:
Im not sure tagging owners for individual pages is actually useful or what problem it's solving. It's a bit too granular. Is it necessary? Given that each page has a "contributors" section when you view it?

Also, whenever we add a new page, we'll need to remember to add it here too, so I worry this will get out of date quick

Here's an alternative of what we can do:

I imagine the usefulness of having this owners page is to know who to tag as reviewers when making changes to these files? If this is the case, can we just say "tag the people who contributed to it" ?

Reply:
The only thing I'm concern about is maintenance. Updating this will be forgotten, so we should keep the categories as broad as possible. 

I feel more comfortable about someone being the owner and first point of contact then bringing other people in where necessary.

Reply:
Works for me